README.md

```diff
@@ -47,22 +47,46 @@ From your shell or command line:
 ### [OPTIONAL] Step 2:  Register the sample on the app registration portal
 
 You can run the sample as is with its current settings, or you can optionally register it as a new application under your own developer account.
-Create a new app at [apps.dev.microsoft.com](https://apps.dev.microsoft.com), or follow these [detailed steps](https://azure.microsoft.com/en-us/documentation/articles/active-directory-v2-app-registration/).  Make sure to:
 
-- Copy down the **Application Id** assigned to your app, you'll need it in the next optional step.
-- Add the **Native Application** platform for your app.
+#### Choose the Azure AD tenant where you want to create your applications
+
+As a first step you'll need to:
+
+1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account or a personal Microsoft account.
+1. If your account is present in more than one Azure AD tenant, select `Directory + Subscription` at the top right corner in the menu on top of the page, and switch your portal session to the desired Azure AD tenant.
+1. In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations (Preview)**.
+
+#### Register the client app (active-directory-xamarin-native-v2)
+
+1. In **App registrations (Preview)** page, select **New registration**.
```

Comment:
Drop the `(preview)` I think as per guidance. Also, I think you're supposed to link to the new blade.